standards/rmrk2.0.0/interactions/equippable.md

```diff
@@ -0,0 +1,57 @@
+# EQUIPPABLE

```

Comment:
No need for `UNEQUIP` just allow to pass empty string instead of `baseslot` to `EQUIP`` instead